pkg/operator2/starter.go

```diff
@@ -74,6 +74,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		"openshift-authentication",
 		"openshift-config",
 		"kube-system",
+		"openshift-config-managed",
```

Comment:
Keep alphabetical order.

You also need to add it here --v. This will cause the observer to resync on every change of a secret/cm in the `openshift-config-managed` NS, but I guess it's still better than us polluting the metrics.

https://github.com/openshift/cluster-authentication-operator/blob/8bb2e80899ee2aae1e7c9a8e8d550be3cd1d602b/pkg/controllers/configobservation/configobservercontroller/observe_config_controller.go#L28-L28

Reply:
> Keep alphabetical order.

you meant reverse alphabetical order? ;) 

Added this namespace to the list you pointed. please take a look